index.html

```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
+            </li>
+        </ul>
+        <button type="button" class="cardbut">
+            See Project
+        </button>
+    </section>
+    <br>
+    <br>
+    <br>
+    <section class="tonic2">
+        <div class="tonicimage">
+            <img src="img/Snapshoot Portfolio3.png" alt="snapshot">
+        </div>
+        <div class="tonichead">
+            <p>Tonic</p>
+        </div>
+        <ul class="toniclists1">
+            <li class="canopy">
+                Canopy
+            </li>
+            <li class="back">
+                Back End Dev  
+            </li>
+            <li class="twenty">
+                2015
+            </li>
+        </ul>
+        <div class="tonictext">
+            <p>
+                A daily section of privately personalized reads; no accounts or sign-ups required
+            </p>
+        </div>
+        <ul class="skills">
+            <li>
+                html
+            </li>
+            <li>
+                CSS
+            </li>
+            <li>
+                JavaScript
+            </li>
+        </ul>       
+            <button type="button" class="cardbut">
+                See Project
+            </button>
+    </section>
+    <br>
+    <br>
+    <br>
+    <section class="multi2">
+        <div class="tonicimage">
+            <img src="img/Snapshoot Portfolio4.png" alt="snapshot">
+        </div>
+        <div class="tonichead">
+            <p>Multi-Post <br> Stories</p>
+        </div>
+        <ul class="toniclists1">
+            <li class="canopy">
+                Canopy
+            </li>
+            <li class="back">
+                Back End Dev  
+            </li>
+            <li class="twenty">
+                2015
+            </li>
+        </ul>
+        <div class="tonictext">
+            <p>
+                A daily section of privately personalized reads; no accounts or sign-ups required
+            </p>
+        </div>
+        <ul class="skills">
+            <li >
+                html
+            </li>
+            <li>
+                CSS
+            </li>
+            <li>
+                JavaScript
+            </li>
+        </ul>
+        <button type="button" class="cardbut">
+            See Project
+        </button>
+    </section>
+    <br>
+    <br>
+
+    <section class="about">
+        <div class="middle">
+            <div class="text1">
+                <h2>About <br> Myself</h2>
+            </div>
+            <div class="text2">
+                <p>
+                    I am an Information and Communication Engineer and full stack website developer. I can help you build a product, feature or website look throug some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. 
+                </p>
+            </div>
+           <br>
+           <br>
+            <div class="connect">
+                <div class="text3">
+                    <p>LET'S CONNECT </p>
```

Comment:
- [ ] Kindly use header tag as well for this title. 😄

Reply:
- [ ] Kindly change the p tag by a header tag